.github/workflows/advanced.yml

```diff
@@ -376,26 +376,45 @@ jobs:
       runtime-package-name: ${{ env.PACK_TOOLCHAIN == 'true' && env.RUNTIME_PACKAGE_NAME || '' }}
       runtime-artifact-name: ${{ env.PACK_TOOLCHAIN == 'true' && env.RUNTIME_ARTIFACT_NAME || '' }}
       toolchain-cache-key: ${{ env.PACK_TOOLCHAIN == 'true' && steps.cache-keys.outputs.toolchain-cache-key || '' }}
```

Comment:
When `aarch64-pc-cygwin-msvcrt-toolchain-cache-key` and `aarch64-pc-cygwin-msvcrt-toolchain-cache-key` are used the `toolchain-cache-key` should not be needed. The same for `*-runtime-cache-key` and for `workflow_call: outputs:` .

Reply:
Sorry, if confusing in regards to the previous comment: The `worklfow_call: outputs: ` , especially the `toolchain-package-name` and  `runtime-package-name` needs to be refactored in the same manner as `*-runtime-cache-key` here. In other words, the  `workflow_call: outputs: runtime-package-name` needs to be renamed to `workflow_call: outputs: aarch64-w64-mingw32-msvcrt-runtime-package-name` because in the current state of the `advanced.yml`, the `workflow_call: outputs: toolchain-package-name` would be set to a value of the job that finishes last causing issue in `rebase.yml`.

Reply:
Could you please clarify why runtime-package-name is needed as output for workflow_call on rebase?

Reply:
It creates a relase [https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/releases](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/releases) from those artifacts.

Reply:
I think it might be removed from output and used as expected output.

Reply:
I suggest this change
```
.github/workflows/rebase.yml
@@ -266,6 +266,8 @@ jobs:
         uses: softprops/action-gh-release@v2
         with:
           files: |
-            ${{ needs.build.outputs.toolchain-package-name }}
-            ${{ needs.build.outputs.runtime-package-name }}
+            aarch64-w64-mingw32-msvcrt-toolchain.tar.gz
+            aarch64-w64-mingw32-msvcrt-runtime.tar.gz
+            aarch64-pc-cygwin-msvcrt-toolchain.tar.gz
+            aarch64-pc-cygwin-msvcrt-runtime.tar.gz
           tag_name: ${{ env.TAG }}
```

Reply:
Anyway, it will not be broken. Release can be adjusted in a separate PR.

Reply:
I am OK with the proposed change. I don't think we need to be relasing Cygwin toolchain as for now. I'd also with this change delete the `workflow_call: outputs:` completely - the values are currently broken and we might forget to fix it in the future.

Reply:
I will create a separate PR to remove outputs and adjust rebase.yml

Reply:
Are you going to integrate it before this one or after? I other words, do you need an approval on this now or then?

Reply:
rebase will be updated later after merging this PR.